src/main/java/org/mdbenefits/app/cli/GoogleDriveClientImpl.java

```diff
@@ -124,11 +149,33 @@ public List<File> findDirectory(String name, String parentId) {
         return List.of();
     }
 
+    @Override
+    public boolean trashDirectory(String name, String directoryId, Map<String, String> errors) {
+        try {
+            File metadata = new File();
+            metadata.setTrashed(true);
+            service.files()
+                    .update(directoryId, metadata)
+                    .setSupportsTeamDrives(true)
+                    .setSupportsAllDrives(true)
```

Comment:
It seems unobvious that we would set these when trashing a directory. Can you add a comment about why this is necessary?

Reply:
I need to double check that this is necessary here. The only reason I include these is that when we are accessing prod it's a Team drive and requires these be set.  The staging folder is not a team drive and doesn't need these. 

I don't quite know the boundaries of when these are needed and when they are not, so I'll double check it here. 

Does seem to be needed to access data.

Reply:
So it appears that if I don't include this I don't get access to the Team drive environment.  Even in the context of trashing a directory.